modules/internal/String.chpl

```diff
@@ -1094,9 +1094,9 @@ module String {
     /*
      Checks if all the characters in the string are printable. Characters are
      defined as being printable if they are not within the range of `0x00-0x1f`
-     or are `0x7f`.
+     and are not `0x7f`.
```

Comment:
I'd probably prefer a positive form of the printable range; that is "defined as being printable if they are within the range 0x20 - 0x7e, including the bounds"